### DISCUSSION TITLE
Welcome to ATAK-Maps Discussions!

### DISCUSSION THREAD

**Discussion by joshuafuller** [joshuafuller/ATAK-Maps#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by githubuser-sketch** [comment]
In regards to the way these maps work, do they send location data to the host it is pulling from (such as Google, Bing, etc)? Or does it just pulling map info?

**Reply by joshuafuller** [reply]
ATAK does not send your precise location data to map service providers like Google or Bing. Instead, it requests map tiles for the area you are viewing or have locked on your screen, which might give the impression of location sharing. Additionally, ATAK caches these map tiles locally on your device, and reuses them from the local cache when you view the same area and zoom level again, minimizing further data requests to the map service.

**Reply by githubuser-sketch** [reply]
Thanks for the clarification!

**Comment by Beartooth69** [comment]
Hey  Josh, I am am a new user of ATAK and Beartooth Mesh systems. I've been reviewing as much information as I can absorb from YouTube etc including T Rex Labs of course. I freely confess I am an old guy and therefore tech handicapped (no handicap tag for my truck though) so I am trying to learn as I go. I am a SWAT operator for a regional team and I can see the advantages to the encrypted mesh network on ops. I am having difficulty downloading and incorporating your open source maps on to the Android phone and further I am trying to understand/figure out how to incorporate sketched up floor plans of buildings into the map layering. Hoping for some guidance even if it's directing me to another video tutorial!

**Reply by tim292stro** [reply]
> ...I am trying to understand/figure out how to incorporate sketched up floor plans of buildings into the map layering. Hoping for some guidance even if it's directing me to another video tutorial!...

Probably the easiest way to add building floor plans is to do Georeferenced PDFs.  Would make deploying a data package for the given building (or campus) fairly "simple" (I understand your form of handicap).  The data package would be a map file PDF for each floor, and you'd turn them on and off as layers in the ATAK UI.
  
QGIS is a Free-and-Open-Source-Software (FOSS) tool that you can play with at home (before you bring this to the big guys who sign checks).  There are  many tutorials for creating georeferenced PDFs using QGIS.  Here are just a few from some quick Google searching:
  
Web tutorial: https://support.foreflight.com/hc/en-us/articles/360036971113-How-do-I-create-a-geospatial-georeferenced-PDF
Youtube Tutorial: https://www.youtube.com/watch?v=jKLBFddpTGI

Good luck and godspeed, they don't pay you guys enough.

**Reply by Beartooth69** [reply]
Thank you sir! I'm trying to get a couple of the younger guys to sit down with me that will probably understand all of this better than me but really appreciate the help. May circle back to you with questions if that's alright or if we figure something out so you can spread the "how to" of it to other teams.

**Comment by NEUprogrammer** [comment]
Hey josh,I am a student.May I ask if ATAK can currently load vector tile maps? Specifically, those in the .pbf format based on the MVT standard?

**Comment by coalkoyote** [comment]
Hello All,
I am not technically well versed, so this is all a little difficult for me to navigate. I am trying to add an NC Property map from their official website. I have been able to successfully link their Polygon map to ATAK however am unable to see ownership, only the outline. Does anyone know the way, if there is one, to view the outline and name of property ownership?
Thanks
